controllers/withwatcher/suite_with_watcher_test.go

```diff
@@ -79,7 +79,7 @@ const (
 	webhookChartPath       = "../../skr-webhook"
 	istioResourcesFilePath = "../../config/samples/tests/istio-test-resources.yaml"
 	virtualServiceName     = "kcp-events"
-	gatewayName            = "lifecycle-manager-kyma-gateway"
+	gatewaySelector        = "operator.kyma-project.io/gateway=default"
```

Comment:
could we use the constants available in v1alpha1 package instead of hardcoding the values?

Reply:
The reason is primarily my lack of knowledge of the v1alpha1 package :) In addition gatewayName was hardcoded here before. I'll define a new constant in the v1alpha1 with a default value for this flag.